purchase_package_qty/__manifest__.py

```diff
@@ -40,7 +40,7 @@
         'views/purchase_order_view.xml',
         'views/stock_picking_view.xml',
         'views/account_invoice_view.xml',
-        'views/report_stockinventory.xml',
+        # 'views/report_stockinventory.xml',
```

Comment:
you're removing this view.

But also, why are we changing `purchase_package_qty`? that's not in the scope of this PR and that module is supposed to be working :thinking: